### ISSUE TITLE
Android Imagetracking db (.imgdb) and image anchor scaling support

### ISSUE THREAD

**Issue by nickolanack** [EddyVerbruggen/nativescript-ar#66]
@EddyVerbruggen  
Good news...
converting a folder of images into an .imgdb is relatively easy. Android provides a cmd line tool https://developers.google.com/ar/develop/c/augmented-images/arcoreimg and its much faster to load. So i think this should be the preferred method 

we can also provide the measured width of images (similar to ios) which greatly speeds up the image tracking time! (btw. we can also specify the width when adding images manually or from folder..) - this also seems to have the effect of scaling the local dimensions of the augmented image so that adding/positioning nodes can be done consistently (just like ios is working atm)

so the scaling factor that I introduced should be removed! yay!
https://github.com/EddyVerbruggen/nativescript-ar/blob/dd6983c71de13be99b4ca3cf9fd29e237ccdc7c1/src/ar.android.ts#L420-L424

**Comment by nickolanack** [comment]
However it will still be possible to provide tracking assets without defined width (and likely to occur)- and in that case the scale factor becomes relevant again but I was thinking that I would remove the scaling option anyway, and instead provide the image width and height in ARTrackingImageDetectedEventData as well as addNode method which a user could use scale themselves if they wanted to...

edit: just noticed that ARTrackingImageDetectedEventData already expects size! lol

**Comment by nickolanack** [comment]
for reference here are the commands i used to build .imgdb for pokemon app
```bash

#build from folder
./arcoreimg build-db --input_images_directory=../demo-pokemon/App_Resources/Android/src/main/assets/PokemonResources/ --output_db_path=../demo-pokemon/App_Resources/Android/src/main/assets/PokemonResources/

#modify -imglist.txt and fix names and append '|0.05' to set width 
#rebuild from list
./arcoreimg build-db --input_image_list_path=../demo-pokemon/App_Resources/Android/src/main/assets/PokemonResources/-imglist.txt --output_db_path=../demo-pokemon/App_Resources/Android/src/main/assets/PokemonResources/
```

**Comment by nickolanack** [comment]
Let me know if you have any thoughts on this. I'll PR .imgdb support shortly
